### ISSUE TITLE
🚀 | Kopfhörerausgang (variabel) 

### ISSUE THREAD

**Issue by deheingit** [MiczFlor/RPi-Jukebox-RFID#786]
Hallo zusammen,

ich bastel gerade eine Phoniebox und hätte eine Frage/Vorschlag:

- Ich würde die Box mit einem miniamp bauen und dort direkt die Lausprecher anschließen
--> Was apssiert wenn ich dann einen Kopfhörer in die Buchste stecke? ich vermute nichts, da das ALSA Device fest eingetragen ist, oder?

--> Wäre es möglich die Phoniebox so zu verwenden, dass bei eingestecktem Kopfhörer (3,5mm) auf diesen umgeschaltet wird (wie man es auch von anderen Geräten kenn) und beim abziehen wieder zurück

VG Dennis

**Comment by Yordan1976** [comment]
Ich habe diese Kopfhörerbuchsen verwendet: https://de.aliexpress.com/item/32847629394.html?spm=a2g0s.9042311.0.0.27424c4dLXIJ5o
Die unterbrechen dann das Signal zu den Lautsprechern mechanisch, solange was eingesteckt ist. Aber auch hier geht es wohl ohne MiniAmp einfacher.

**Comment by Yordan1976** [comment]
So sieht das Konstrukt aus:
![grafik](https://user-images.githubusercontent.com/35901879/74191563-23cfb480-4c54-11ea-9e78-a48832332ae0.png)

**Comment by deheingit** [comment]
Hi @Yordan1976: Danke. 
Das heißt bei dir hat das Umschalten eigentlich nichts mit dem rpi zu tun sondern ist einfach von der USV Soundkarte gesteuert...

**Comment by Yordan1976** [comment]
Nein, die Soundkarte steuert da gar nichts, die gibt nur das Signal aus. Das Umschalten passiert ganz alleine durch das mechanische Öffnen/Schließen der Kontakte in der Buchse.

**Comment by ZyanKLee** [comment]
solche Buchsen habe ich auch genommen für diesen Zweck. Funktioniert auch sehr gut.

**Comment by domu83** [comment]
@Yordan1976 ... hab mir die Seinerzeit auch bestellt, aber noch nicht verbaut. Hast Du evtl die Pinbelegung zur Hand? Ist mir nicht ganz ersichtlich, aber evtl bin ich auch einfach doof, nicht auszuschließen 🙄

**Comment by Yordan1976** [comment]
Da habe ich mir doch tatsächlich gute Notizen zu gemacht. Bitte sehr!
![ScreenShot 106 Issues · MiczFlor_RPi-Jukebox-RFID - Mozilla Firefox](https://user-images.githubusercontent.com/35901879/74610152-db4f4580-50f0-11ea-9732-6602b04c3389.png)

**Comment by domu83** [comment]
@Yordan1976  Der Hammer, Du bist ja cool! fecht herzlichen Dank!!

**Comment by mimi-ac** [comment]
Hallo,
kann es sein, dass oben beschriebene Anschluss nicht in Kombination mit dem Hifiberry Miniamp funktioniert?

**Comment by deheingit** [comment]
Hi @mimi-ac: Ich habe es auch so verstanden, dass der Stecker nur mit aktiven lautsprechern funktioniert. Der Minimp ist ja ein DAC, an den dann passive Lautsprecher angschlossen werden. z.B. normale Kopfhörer oder Aktivboxen kann man meines wissens nicht an den miniamp anschließen

**Comment by domu83** [comment]
@Yordan1976 ich hatte mich am WE mal wieder an dem Thema versucht, allerdings bin ich gescheitert... Offensichtlich bin ich zu doof selbst mit Deiner Verkabelungsinfo das hinzubekommen.
Welche Lautsprecher hast Du denn genommen?

**Comment by Yordan1976** [comment]
Diese Speedlink Twoxo, weil die auch kein störendes Rauschen haben:
https://smile.amazon.de/gp/product/B01HDR5EIK/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&psc=1
Aber im Prinzip hast du ja einfach drei Leitungen (Signal L + R und GND), von denen die Buchse die Signalleitungen beim Einstecken mechanisch trennt.

**Comment by domu83** [comment]
danke Dir.... irgendwie hab ich in meiner Konstellation mit den Logitechboxen und mit dem Kabel scheinbar einen kurzschluss erzeugt. Next try...

**Comment by manajoe** [comment]
@Yordan1976 Ich hab deine Anleitung mit den Trust-Boxen ([https://www.amazon.de/Trust-Lautsprecher-USB-Stromversorgung-Smartphone-schwarz/dp/B00JRW0M32/]) umgesetzt und noch eine Frage, wie du das mit der Lautstärke-Regelung gelöst hast.
Ich nutze eine USB-Soundkarte und habe bisher am Drehregler der Lautsprecher die Lautstärke runtergeregelt und im Gehäuse versteckt. Lauter/Leiser mache ich dann über die Phoniebox VolumeControl, von 0 bis 100%.
Wenn ich jetzt einen Kopfhörer an die zwischengeschaltete Buchse anstecke, ist der Ton unfassbar laut, der Drehregler der Lautsprecher greift hier ja nicht. Um eine erträgliche Lautstärke zu bekommen, müsste ich softwareseitig MAXVOL auf viell. 30% einstellen, dafür dann den Drehregler der Lautsprecher hochdrehen, damit ich dort wieder was höre.
Oder hast du das anders gelöst?

Danke und Grüße
Manajoe

**Comment by Yordan1976** [comment]
Ich muss zugeben, dass der Kopfhörerausgang bisher noch nicht genutzt wird und mir das Problem daher nicht aufgefallen ist. Ich gehe aber davon aus, dass es nicht anders sein wird. Aber dein Weg ist prinzipiell nicht falsch. Die Boxen habe ich auch so eingestellt, dass sich bei 100% nicht die Nachbarn beschweren. Es wäre schön, wenn man softwareseitig feiner regeln könnte. Ich glaube erst ab 3% wird überhaupt etwas ausgegeben und wenn man die Boxen am Poti lauter dreht, ist das schon nicht so leise, wie ich es in ruhiger Umgebung gerne hätte.

**Comment by vanMacG** [comment]
Auch wenn das Thema schon etwas älter ist wollte ich kurz meine Erfahrungen teilen.
Die kleine Kopfhörerbuchse gibt es auch bei Reichelt: [EBS 35](https://www.reichelt.de/klinkeneinbaubuchse-3-5-mm-stereo-ebs-35-p7301.html)
Was bei der Pin-Belegung von @Yordan1976 noch zu erwähnen ist: An Pin 1 = Masse muss das Massesignal von der Soundkarte (Audio-Quelle) und der Lautsprecher angelötet werden. D.h. es sind dort dann zwei Litzen angelötet, an den anderen Pins nur eine Litze.
Bzgl. Lautstärke: Ich verwende diese günstigen Lautsprecher von [nedis](https://www.reichelt.de/lautsprecher-pc-laptop-stereo-3-5-mm-buchse-n-cspr10020bk-p242512.html). Als Audio-Quelle verwende ich direkt den Klinkenausgang vom Raspi. Wenn ich den Lautstärkeregler der Lautsprecher auf max drehe (ist am Ende in der Box versteckt), ist die Lautstärke relativ gut mit dem Kopfhörer abgeglichen. Die Gesamtlautstärke regle ich dann über die Software. Die Abstimmung war bei mir direkt ziemlich gut imho.
